public/docs/getting_started/migrations.md

```diff
@@ -172,6 +172,16 @@ Deploy a specific contract, specified by the contract object, with optional cons
 
 You can optionally pass an array of contracts, or an array of arrays, to speed up deployment of multiple contracts. Additionally, the last argument is an optional object that observes a single key, `overwrite`. If `overwrite` is set to `false`, the deployer won't deploy this contract if one has already been deployed. This is useful for certain circumstances where a contract address is provided by an external dependency.
 
+```javascript
+deployer.deploy(Migrations, {overwrite: false});
+```
+
+The options can also be used to specify the parameters when sending the transaction:
```

Comment:
Would you mind updating the line above that mentions a “single key”, since `gas` and `from` are also accepted?

Reply:
@bmmpxf I'd reword this as the parameters have already been referenced, then link out to somewhere else if the reader wants more information. I did a quick search of the Truffle docs and couldn't find an example, so perhaps link to the `truffle-contract` readme.